lmdeploy/turbomind/tokenizer.py

```diff
@@ -95,6 +120,8 @@ def __init__(self, model_dir: str):
                   'It may take long time to initialize the tokenizer.')
         self.model = AutoTokenizer.from_pretrained(model_dir,
                                                    trust_remote_code=True)
+        self.need_padding = isinstance(self.model, LlamaTokenizerFast)
```

Comment:
Do we only support llama in huggingface model? Or is it better to have a list of "need_padding" tokenizers?

Reply:
Other tokenizer can automatically pad white space.